app/routes/insertEntry.tsx

```diff
@@ -12,6 +14,12 @@ export async function action({ request }: ActionArgs) {
   return redirect(`/entries/${data.headword}`)
 }
 
+export async function loader({ request }: LoaderArgs) {
+  await redirectIfUserLacksPermission(request, "det:createDraft")
```

Comment:
So I think we haven't set up permissions for me here yet, so I can't test this part. But the redirect works at least haha

Reply:
You should now have admin privileges. You might need to log out (go to the Admin page to do this if necessary) and log back in. Let me know how it goes!

Reply:
The access works by the way. Thanks!